CHANGELOG.md

```diff
@@ -15,14 +15,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Change the meaning of the `fee_amount` field inside `CoinSelectionResult`: from now on the `fee_amount` will represent only the fees asociated with the utxos in the `selected` field of `CoinSelectionResult`.
 - New `RpcBlockchain` implementation with various fixes.
 - Return balance in separate categories, namely `confirmed`, `trusted_pending`, `untrusted_pending` & `immature`.
+- Add `allow_dust` method on `TxBuilder`.
```

Comment:
You should rebase and move this under the new unreleased section

Reply:
Hey @terror, sorry for the ping, but the feature freeze is tomorrow, which means that if you fix this comment by tomorrow we can merge the PR and release `allow_dust` as a part of bdk 0.22! :)